### PR TITLE
added release notes for 10.18.0.245

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-245.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+<td>Cockpit</td>
+<td>The data export feature in the data explorer has been improved. Now only active data points are included. Moreover, the name of the exported file includes the name of the series. This file contains all measurements in which this series is found.</td>
+<td>MTM-50586</td>
+<td>10.18.0.162</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>An issue with module federation has been fixed. If the shell application was built on a windows machine the modules have not been shared properly between shell and plugin.</td>
```
Comment (rssag) [inline content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-245.md:55]:
```suggestion
<td>An issue with module federation has been fixed. If the shell application was built on a Windows machine the modules have not been shared properly between shell and plugin.</td>

```

File: content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-245.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+<td>Cockpit</td>
+<td>Fixed an issue with SVG files in the SCADA widget, where <code>animate</code> or <code>animateTransform</code> tags worked fine in the configuration preview, but did not work in the dashboard view.</td>
+<td>MTM-55990</td>
+<td>10.18.0.160</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>In rare cases, when a real-time connection was interrupted, certain widgets, for example, did not update again once the connection was re-established. This behavior has been fixed.</td>
```
Comment (rssag) [inline content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-245.md:71]: i found real-time as well as real time (and realtime) in c8y-docs. which one is it?
also, lots of commas due to "for example" - could maybe made be fewer

File: content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-245.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+<td>Cockpit</td>
+<td>The width for the timestamp in the "Event list" widget has been slightly increased.</td>
+<td>MTM-55803</td>
+<td>10.18.0.146</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Web SDK</td>
+<td>Fixed an issue with <code>c8ycli locale-extract</code> on Windows where the extracted <i>locales.pot</i> file didn´t contain any entries.</td>
```
Comment (rssag) [inline content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-245.md:95]:
```suggestion
<td>Fixed an issue with <code>c8ycli locale-extract</code> on Windows where the extracted <i>locales.pot</i> file didn't contain any entries.</td>

```

File: content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-245.md (hunk 2 above)
Comment (BeateRixen) [reply content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-245.md:72]: real-time is the adjective while realtime is the noun. I try to use it quite consistent, following this rule :-).

File: content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-245.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+<td>Cockpit</td>
+<td>Fixed an issue with SVG files in the SCADA widget, where <code>animate</code> or <code>animateTransform</code> tags worked fine in the configuration preview, but did not work in the dashboard view.</td>
+<td>MTM-55990</td>
+<td>10.18.0.160</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>In rare cases, when a real-time connection was interrupted, certain widgets, for example, did not update again once the connection was re-established. This behavior has been fixed.</td>
```
Comment (BeateRixen) [inline content/release-10-18-0/applications-10-18-0-bundle/applications-10-18-0-245.md:72]:
```suggestion
<td>In rare cases, when a real-time connection was interrupted, certain widgets did not update again once the connection was re-established. This behavior has been fixed.</td>

```

**Pull request merged**